### ISSUE TITLE
Semi OT: How to enable non-cloud mode?

### ISSUE THREAD

**Issue by giggls** [osresearch/hcpy#50]
After I figured out that host variables in devices.json must be a resolvable hostname I immediately succeeded running this code on Debian 12.
However I did not yet find out how I can disable the connection from the devices to the cloud. Is this still possible in recent versions of the app?

**Comment by matmai** [comment]
Yes, it is still possible to disable the connection in the cloud in recent versions (tested with 11.6.0). In the app go to the appliance device settings, in network sections select details. There is the switch to disable connection to server.
Then you applicance should be just connected to you phone/app: 

![Image](https://github.com/user-attachments/assets/74983c1f-dfda-49f2-9d4b-a638ddf6742a)

**Comment by dekiesel** [comment]
Can you cut off access at the router level to be sure? Or will that mess with things?


I have a vlan that has no access to the internet. That'd be perfect.

**Comment by matmai** [comment]
> Can you cut off access at the router level to be sure? Or will that mess with things?

It should work without internet, since the cloud does not get any updates.
